Library/src/lab1/Library.java

```diff
@@ -2,48 +2,67 @@
 
 /**
  * A representation of a library of books.
- * @author <a href="mailto:rkhatchadourian@citytech.cuny.edu">Raffi Khatchadourian</a>.
+ *
+ * @author mingjiehuang 
  */
 public class Library {
-    // TODO: Add the missing implementation to this class
 
-    public static void main(String[] args) {
+    // Create two libraries 
+    String address;
+    java.util.ArrayList<Book> books;
+
+    public static final String openingTime = "9am";
```

Comment:
Can be `private` since no other class uses them. Also, should be `OPENING_TIME`.